web/src/app/services/AlertMetrics/AlertMetrics.tsx

```diff
@@ -250,7 +220,10 @@ export default function AlertMetrics({
             <AlertMetricsFilter now={now} />
             <AlertCountGraph data={graphData} />
             <AlertMetricsTable
-              alerts={alertsData.alerts}
+              alerts={alertsData.alerts.map((a) => ({
+                ...a,
+                ...a.metrics,
+              }))}
```

Comment:
is this doing anything since metrics is already on `a`?

Reply:
yes, we're flattening metrics onto one object since the table can't deal with nested properties